pkg/cloudprovider/external_test.go

```diff
@@ -156,6 +156,22 @@ func TestIsCloudProviderExternal(t *testing.T) {
 			},
 		},
 		expected: true,
+	}, {
+		name: "FeatureSet: CustomNoUpgrade (With External Feature Gate), Platform: IBMCloud",
```

Comment:
Since IBM doesn't have an in-tree provider, I would have expected IBMCloud to always report as external rather than depending on a feature gate, WDYT?

Reply:
That makes sense to me. I was basing this off of the example of some of the other providers. The code has been updated.

Reply:
The name of this test should be updated since the feature gates are empty.

Reply:
Updated